src/static.ts

```diff
@@ -37,14 +37,32 @@ export async function getExportsStatic(name: string, options?: GetExportsOptions
     return await map.get(url)!
   }
 
+  async function tryReadFile(path: string) {
+    for (const prefix of ['', '/index']) {
+      for (const ext of ['', '.mjs', '.js']) {
+        try {
+          const suffix = prefix + ext
+          return {
+            code: await fs.readFile(path + suffix, 'utf8'),
+            suffix,
+          }
+        }
+        catch {}
+      }
+    }
+  }
```

Comment:
This feels a bit hacky and less performant, I'd expect a better resolution algorithm.

Reply:
use mlly's resolve to resolve the real path.